gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/authentication/controller/LoginController.java

```diff
@@ -113,7 +129,7 @@ public ModelAndView login(HttpServletRequest request, @RequestParam(value=ORGANI
                 String identityId = identity.getId();
                 SocialAuthenticationProvider socialAuthenticationProvider = (SocialAuthenticationProvider) identityProviderManager.get(identityId);
                 if (socialAuthenticationProvider != null) {
-                    final Maybe<Optional<Request>> maybe = socialAuthenticationProvider.asyncSignInUrl(buildRedirectUri(request, identityId), RandomString.generate()).map(Optional::ofNullable);
+                    final Maybe<Optional<Request>> maybe = socialAuthenticationProvider.asyncSignInUrl(buildRedirectUri(request, identityId), new JWT(), this::processState).map(Optional::ofNullable);
```

Comment:
**TODO**: can we at least initialize the iat and exp claim in this state ?

Reply:
The state JWT we create on the gateway side ([LoginSocialAuthenticationHandler](https://github.com/gravitee-io/gravitee-access-management/blob/b54431c58205fe9f9c2a33d2939a45eccb769754/gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/java/io/gravitee/am/gateway/handler/root/resources/handler/login/LoginSocialAuthenticationHandler.java#L200)) doesn't have `iat` and `exp` claims. SHould we add them there as well?

Should the expiration time be hardcoded (I'm thinking 5 minutes would be a good value), or do we want users to set it via OIDC provider's configuration?